src/client.ts

```diff
@@ -244,6 +245,7 @@ import { ImageInfo } from "./@types/media.ts";
 import { Capabilities, ServerCapabilities } from "./serverCapabilities.ts";
 import { sha256 } from "./digest.ts";
 import { keyFromAuthData } from "./common-crypto/key-passphrase.ts";
+import { RustCryptoEventMap } from "./rust-crypto/rust-crypto.ts";
```

Comment:
:-1: please don't import from rust-crypto here: it breaks the dynamic loading of the rust crypto library.

Add a new CryptoEventMap to `crypto-api` and use that instead.